tests/test_openlibrary.py

```diff
@@ -113,23 +146,55 @@ def test_edition_validation(self):
                                       'value': '2016-10-12T00:48:04.453554'
                                   },
                                   authors=[author])
-        #edition = self.ol.Edition.get(u'OL2183333M')
         self.assertIsNone(edition.validate())
-        #orphaned_edition = self.ol.Edition.get(u'OL17922113M')
         orphaned_edition = self.ol.Edition(edition_olid='OL123M',
                                   work_olid=None,
                                   title='Test Title',
                                   authors=[author])
         with self.assertRaises(jsonschema.exceptions.ValidationError):
             orphaned_edition.validate()
 
-    def xtest_cli(self):
-        #TODO: re-write this test once edition loading becomes stable. Avoid live requests.
-        expected = json.loads("""{"subtitle": "a modern approach", "series": ["Prentice Hall series in artificial intelligence"], "covers": [92018], "lc_classifications": ["Q335 .R86 2003"], "latest_revision": 6, "contributions": ["Norvig, Peter."], "py/object": "olclient.openlibrary.Edition", "edition_name": "2nd ed.", "title": "Artificial intelligence", "_work": null, "languages": [{"key": "/languages/eng"}], "subjects": ["Artificial intelligence."], "publish_country": "nju", "by_statement": "Stuart J. Russell and Peter Norvig ; contributing writers, John F. Canny ... [et al.].", "type": {"key": "/type/edition"}, "revision": 6, "last_modified": {"type": "/type/datetime", "value": "2010-08-03T18:56:51.333942"}, "authors": [{"py/object": "olclient.openlibrary.Author", "bio": "", "name": "Stuart J. Russell", "links": [], "created": "2008-04-01T03:28:50.625462", "identifiers": {}, "alternate_names": ["Stuart; Norvig, Peter Russell"], "birth_date": "", "olid": null}], "publish_places": ["Upper Saddle River, N.J"], "pages": 1080, "publisher": ["Prentice Hall/Pearson Education"], "pagination": "xxviii, 1080 p. :", "work_olid": "OL2896994W", "created": {"type": "/type/datetime", "value": "2008-04-01T03:28:50.625462"}, "dewey_decimal_class": ["006.3"], "notes": {"type": "/type/text", "value": "Includes bibliographical references (p. 987-1043) and index."}, "identifiers": {"librarything": ["43569"], "goodreads": ["27543"]}, "cover": "", "publish_date": "2003", "olid": "OL3702561M"}""")
+
+class TestCLI(unittest.TestCase):
```

Comment:
Do we a need a separate class for this? The methods just look like they're testing methods (similar to `test_get_edition_by_isbn`).

Reply:
the subclass is just to group the two tests that share the same expectation and set-up data, one getting by isbn, the other directly by olid

Reply:
Ah, ok. Maybe give it a different name? `TestCLI` is a bit misleading.